packages/apollo-language-server/src/engine/index.ts

```diff
@@ -130,9 +130,9 @@ export class ApolloEngineClient extends GraphQLDataSource {
         );
       }
       // use error logger
-      // if (errors) {
-      //   throw new Error(errors);
-      // }
+      if (errors) {
+        throw new Error(errors.map(error => error.message).join("\n"));
```

Comment:
Can we DRY this up a bit by creating a helper?  This seems like it's going to be a very common pattern that's going to perpetuate if we don't nip it in the bud right now.

At the very least, this might be a `mapGraphQLErrorsToString(errors)` (and be `errors.map(error => error.message).join("\n")`), but I'd even go so far to say we could have a `assertNoGraphQLErrors(errors);` function which would be all three lines here?